repository/src/main/java/org/apache/atlas/repository/store/graph/v2/preprocessor/contract/ContractVersionUtils.java

```diff
@@ -0,0 +1,85 @@
+package org.apache.atlas.repository.store.graph.v2.preprocessor.contract;
+
+import org.apache.atlas.discovery.EntityDiscoveryService;
+import org.apache.atlas.exception.AtlasBaseException;
+import org.apache.atlas.model.discovery.AtlasSearchResult;
+import org.apache.atlas.model.discovery.IndexSearchParams;
+import org.apache.atlas.model.instance.AtlasEntity;
+import org.apache.atlas.model.instance.AtlasEntityHeader;
+import org.apache.atlas.repository.graphdb.AtlasGraph;
+import org.apache.atlas.repository.store.graph.AtlasEntityStore;
+import org.apache.atlas.repository.store.graph.v2.*;
+import org.apache.atlas.type.AtlasTypeRegistry;
+
+import java.util.*;
+
+import static org.apache.atlas.repository.Constants.*;
+import static org.apache.atlas.repository.util.AtlasEntityUtils.mapOf;
+
+
+public class ContractVersionUtils {
+
+    private final EntityMutationContext context;
+    public final EntityGraphRetriever entityRetriever;
+    private final AtlasTypeRegistry atlasTypeRegistry;
+    private AtlasEntityStore entityStore;
```

Comment:
unused - remove

Reply:
Removed the class altogether and added the getLatestVersion to ContractPreProcessor.